### PR TITLE
Add the ability to use up and down keys, better errors with red color…

### PR DESCRIPTION
**Pull request by BishoyHanyRaafat**
I added few thing trying to help and contribute 

1. I edited most of the error messages to show in red 
2. Command suggestion 
3. Now you can use the up and down arrows in the loop/run command
image
image

### CODE REVIEW

File: src/pieces/commands.py [hunk 1]
```diff
@@ -348,6 +349,40 @@ def check():
 ###############################################################################
 ############################## HELPER FUNCTIONS ###############################
 ###############################################################################
+def levenshtein_distance(s1, s2):
```
Comment (shivay-at-pieces) [inline src/pieces/commands.py:352]: probably the helper function should be in a separate file

Comment (shivay-at-pieces) [reply src/pieces/commands.py:352]: Sounds good. But this is a great implementation!

Comment (BishoyHanyRaafat) [reply src/pieces/commands.py:352]: Ty

**Review by sam-at-pieces — APPROVED** [review]

**Comment by sam-at-pieces** [comment]
LGTM, I am happy to merge this, @hal-8999-alpha can we get your ok?

**Pull request merged**